modules/apps/trash/trash-taglib/src/main/resources/META-INF/resources/undo/page.jsp

```diff
@@ -90,8 +90,8 @@ int trashedEntriesCount = GetterUtil.getInteger(request.getAttribute("liferay-tr
 		<aui:input name="redirect" type="hidden" value="<%= redirectURL %>" />
 		<aui:input name="restoreTrashEntryIds" type="hidden" value="<%= StringUtil.merge(restoreTrashEntryIds) %>" />
 
-		<aui:button cssClass="alert-link btn-link btn-sm btn-unstyled trash-undo-button" type="submit" value="undo" />
+		<aui:button cssClass="alert-link btn-link btn-sm btn-unstyled trash-undo-button" primary="false" type="submit" value="undo" />
 	</aui:form>
 </liferay-util:buffer>
 
-<liferay-ui:success key="<%= portletDisplay.getId() + SessionMessages.KEY_SUFFIX_DELETE_SUCCESS_DATA %>" message="<%= alertMessage %>" />
\ No newline at end of file
+<liferay-ui:success key="<%= portletDisplay.getId() + SessionMessages.KEY_SUFFIX_DELETE_SUCCESS_DATA %>" message="<%= alertMessage %>" messageType="html" />
```

Comment:
The main problem with this is that it is a breaking change. We've identified this case, but don't know how many more could be affected. That includes customers which would need to update their code in order to address this.

We should:
- Find a way to make this work as before
- Document the breaking change if inevitable

Could we focus on making this work for both `html` and `non-html` messages

Reply:
I found these alternatives:

* Consider type `html` by default as fixed and fix as you commented https://github.com/liferay-frontend/liferay-portal/pull/173#issuecomment-668035248

* Have a regexp(I know, it's wrong but 💁‍♂️ ) to detect if the given message was a HTML or not

When considering the first alternative, I can place a notice that will be visible when on development mode to warn the developer about this breaking change. What do you think @jbalsas?

Reply:
> When considering the first alternative, I can place a notice that will be visible when on development mode to warn the developer about this breaking change. What do you think @jbalsas?

Could we use `invariant` or `warning` for doing it?